build.rs

```diff
@@ -62,10 +64,13 @@ fn main() -> std::io::Result<()> {
     #[cfg(feature = "serde")]
     {
         config.type_attribute(".", "#[serde(rename_all = \"camelCase\")]");
-        config.type_attribute(".", "#[allow(clippy::doc_lazy_continuation)]");
-        config.type_attribute(".", "#[allow(clippy::empty_docs)]");
     }
 
+    config.type_attribute(".", "#[allow(clippy::doc_lazy_continuation)]");
+    config.type_attribute(".", "#[allow(clippy::empty_docs)]");
+    config.type_attribute(".", "#[allow(missing_docs)]");
+    config.type_attribute(".", "#[allow(clippy::doc_overindented_list_items)]");
```

Comment:
These two are new and required because the generated types don't adhere to our newly required lint.

Reply:
Fixed by [cee7650](https://github.com/meshtastic/rust/pull/52/commits/cee7650c7f5d98289bf3e321c6a42ad87b074ee9).